extensions/applicationinsights-react-native/README.md

```diff
@@ -36,6 +36,12 @@ By default, this plugin automatically collects
  - **Device Model Name** (iPhone XS, etc.)
  - **Device Type** (Handset, Tablet, etc.)
 
+ ## Uncaught Exceptions Collected
```

Comment:
Would the user need to install and link `react-native-exception-handler` like they would need to with the device-info package?

Reply:
yes you are right, added in next update. thanks